producer.py

```diff
@@ -181,6 +183,13 @@ def package_releases_update(
                     package_index.url,
                     only_if_package_seen=only_if_package_seen,
                 )
+                package_releases.publish_to_topic(
```

Comment:
@fridex @pacospace Do you think this is the right way to gather all messages?

Reply:
I think the logic should be similar to https://github.com/thoth-station/graph-refresh-job/blob/master/producer.py#L114 so the method need to be async e.g.
```
from thoth.messaging import MessageBase

app = MessageBase().app

@app.command()
async def main() -> None:
```

Reply:
So, then maybe I can make [cli](https://github.com/thoth-station/package-releases-job/blob/master/app.py#L271) function to be `async`.

Reply:
How about this https://github.com/thoth-station/package-releases-job/pull/449/commits/53be2079bae6dde82b5d149cce0226c22c0d622b?